### PR TITLE
Windows: update to quantlib 1.29

### PR DESCRIPTION
**Pull request by jeroen**
I'm guessing you're going to need this to get your next release past the cran winbuilder as well.

### CODE REVIEW

File: src/Makevars.win [hunk 1]
```diff
@@ -6,19 +6,10 @@
 # Copyright 2011         Uwe Ligges, Brian Ripley, and Josh Ulrich
 # Copyright 2018 - 2019  Jeroen Ooms
 
-RWINLIB=../windows/quantlib-1.16
+RWINLIB=../windows/quantlib-1.29
```
Comment (eddelbuettel) [inline src/Makevars.win:9]: That is awesome.  Any reason not to have taken last month's 1.30 release?   (1.29 should suit us for a while.)

**Review by eddelbuettel — APPROVED** [review]
Fingers crossed it passes CI ...

File: src/Makevars.win (hunk 1 above)
Comment (jeroen) [reply src/Makevars.win:9]: I copied from msys2 https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-quantlib/PKGBUILD

**Comment by jeroen** [comment]
You don't have any Windows CI, right? But I guess you can upload to winbuilder.

**Comment by eddelbuettel** [comment]
Or r-universe :)

**Comment by jeroen** [comment]
OK, all ✅ now on r-universe.

**Comment by eddelbuettel** [comment]
Big thank you!

**Pull request merged**